README.md

```diff
@@ -106,3 +106,23 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
 
 * Fry
 * Kestrel
+
+<details>
+     <summary>## Included shell scripts (Click to expand)</summary>
```

Comment:
```suggestion
## Included shell scripts
```
The formatting here is a little funky, so let's try it back to the way you originally had it!